src/SonarScanner.Shim/Resources.resx

```diff
@@ -241,4 +241,7 @@ Possible causes:
   <data name="ERR_ProjectBaseDirDoesNotExist" xml:space="preserve">
     <value>The project base directory doesn't exist.</value>
   </data>
+  <data name="WARN_TestResultsNotFound" xml:space="preserve">
+    <value>Test results (.trx) does not exist at the specified location. Path: {0}</value>
```

Comment:
Is it one file or multiple?
For 1 say "Test results file (*.trx) does not"
For 2+ say "Test results files (*.trx) do not"